glean-core/ios/Glean/Dispatchers.swift

```diff
@@ -65,6 +75,11 @@ class Dispatchers {
     /// recording of metrics and things that need to execute in order. Since this executes the tasks on
     /// a non-concurrent (serial) queue, the tasks are executed in the order that they are launched.
     ///
+    /// **Important Note:** This queue should only be used for performing API functions, and not for
```

Comment:
what are "API functions"? 
IMO that's too vague, this whole thing is an API (`Dispatchers.API`) making it more confusing.

Reply:
What about something along these lines:

> **Note**: Tasks that should be processed in order and finish before successive tasks are run should be launched using `launchAPI`. That includes all metric recording functions.

And then on the `launchConcurrent` one something like

> **Note**: Tasks that can be processed asynchronously and can block for a long time, but which should not block other tasks, such as recording data, should be launched using `launchConcurrent`. That for example includes an upload task.

(ok, that first sentence is too long now, maybe you can come up with a better version?)

Reply:
These are good suggestions, let me see what I can do with this...